x-pack/plugins/cloud_security_posture/public/components/subscription_not_allowed.tsx

```diff
@@ -34,12 +41,12 @@ export const SubscriptionNotAllowed = ({
                 defaultMessage="To use these cloud security features, you must {link}."
                 values={{
                   link: (
-                    <EuiLink href={licenseManagementLocator}>
+                    <SubscriptionLink subscriptionContext={subscriptionContext}>
```

Comment:
Hey @janmonschke, I got a question regarding this change

We were recently asked to [verify the availability of the license management plugin](https://github.com/elastic/kibana/issues/153016) before rendering the link in this subscription prompt ([PR](https://github.com/elastic/kibana/pull/157161)). 

Should we keep verifying the availability of the locator before rendering the link after this change? do we need to verify the availability of any other components? 
or will it be available on all deployment types and we can remove the checks completely?

Reply:
Hey @JordanSh ! From how I understand it, the verification of the locator can remain where it is now.